lib/assigner.rb

```diff
@@ -213,7 +213,8 @@ def forbidden_reasons(assign_to:, type:, note:, status:)
       assign_to.is_a?(User) ? :forbidden_assign_to : :forbidden_group_assign_to
     when already_assigned?(assign_to, type, note, status)
       assign_to.is_a?(User) ? :already_assigned : :group_already_assigned
-    when Assignment.where(topic: topic, active: true).count >= ASSIGNMENTS_PER_TOPIC_LIMIT
```

Comment:
Out of scope for this PR, but seeing this line (the count in particular) made me curious.

<img width="520" alt="Screenshot 2023-01-12 at 11 19 52 AM" src="https://user-images.githubusercontent.com/5259935/211968081-c97ee36f-826d-48dd-8481-027c6b546a42.png">

Maybe we should have a counter cache for this. At the very least we should probably add an index on `topic_id` so we don't have to walk the entire `assignments` table. Maybe we create a #todo for this, WDYT? 🙂

Reply:
Excellent catch! I added migration to add topic_id index